### DISCUSSION TITLE
Future of the product

### DISCUSSION THREAD

**Discussion by vmicrodev** [quay/clair#1288] [Q&A] [answered]
Hi everyone,

Could you kindly please advise:
- If you are planning to keep the Clair product in the future as I see no updates in the product code but only in DB.
- Is there a major update in progress?
-  Are there any plans to make the product fully dockerized?

Thank you very much in advance.
Harvy

**Comment by hdonnay** [comment, marked as answer]
> * If you are planning to keep the Clair product in the future as I see no updates in the product code but only in DB.

I don't know what you mean by this. The project is obviously under active development.

> * Is there a major update in progress?

Beyond v4? Not currently.

> * Are there any plans to make the product fully dockerized?

What do you mean by "fully dockerized?" The software runs in and is distributed in a container.

**Reply by vmicrodev** [reply]
Hi Hank,

Thank you for your reply. However I would like to ask about the future of Clair V2:
- Are we expecting bug fixes in Clair V2?
- Are there any Clair V2 new releases soon?

Thanks,
Harvy

**Reply by hdonnay** [reply]
> * Are we expecting bug fixes in Clair V2?

I'm not sure who you mean by "we," but there's no active development that I'm aware of. The current maintainers are around to accept contributed fixes.

> * Are there any Clair V2 new releases soon?

No, there haven't been any changes since the last release.
